samples/multi-agent/src/main/java/demo/multiagent/application/AgentTeamEvaluatorConsumer.java

```diff
@@ -0,0 +1,67 @@
+package demo.multiagent.application;
+
+import akka.javasdk.annotations.ComponentId;
+import akka.javasdk.annotations.Consume;
+import akka.javasdk.client.ComponentClient;
+import akka.javasdk.consumer.Consumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@ComponentId("agent-team-eval-consumer")
+@Consume.FromWorkflow(AgentTeamWorkflow.class)
+public class AgentTeamEvaluatorConsumer extends Consumer {
```

Comment:
Changed approach to evaluate directly on the workflow state instead of indirectly via session memory events. In this case it could (probably better) be done from the workflow itself, but wanted to illustrate a consumer since evaluations are typically background tasks that only report problems.